src/language.rs

```diff
@@ -101,62 +117,94 @@ macro_rules! define_languages {
 }
 
 define_languages! {
-    { Assembly, "assembly.ascii", "Assembly", vec![Color::Cyan] },
```

Comment:
I was afraid of this additional complexity.
Ideally, these two should be valid.
```
{ Assembly, "assembly.ascii", "Assembly", { Colors { basic_colors: vec![Color::Cyan], true_colors: None } } },
{ C, "c.ascii", "C", vec![Color::Cyan, Color::Blue] },
```
That is, the inclusion of the braces should be optional IMO.